templates/whats-new.php

```diff
@@ -0,0 +1,2186 @@
+                    'description' => '',
+                ],
+                [
+                    'title'       => 'Removed user switch setting from Dokan selling setting, now user switching will work if plugin exists',
+                    'description' => '',
+                ],
+            ],
+            'Fix'         => [
+                [
+                    'title'       => 'Added Vue DateRangePicker library',
+                    'description' => '',
+                ],
+                [
+                    'title'       => 'Added missing param on woocommerce_admin_order_item_headers',
+                    'description' => '',
+                ],
+                [
+                    'title'       => 'Fixed WC mail template overwrite was not working',
+                    'description' => '',
+                ],
+                [
+                    'title'       => 'add call to filter dokan_product_cat_dropdown_args to listing-filter.php (thanks to David Marin)',
+                    'description' => '',
+                ],
+                [
+                    'title'       => 'updated dokan_product_seller_info() function to not to add vendor data if vendor id doesn\'t exists (thanks to David Marin)',
+                    'description' => '',
+                ],
+                [
+                    'title'       => 'Hide Show email address in store settings from store settings page if admin disable this settings from customizer.',
+                    'description' => '',
+                ],
+                [
+                    'title'       => 'Added upgrader to change refund and withdraw database table column',
+                    'description' => '',
+                ],
+            ],
+            'Add'         => [
+                [
+                    'title'       => 'Black Friday promotion 2021',
+                    'description' => '',
+                ],
+            ],
+        ],
+    ],
+    [
+        'version'  => 'Version 3.3.0',
+        'released' => '2021-10-31',
+        'changes'  => [
+            'Improvement' => [
+                [
+                    'title'       => 'Added integration of sweetalert2 for alert, prompt, confirm, toast notification',
+                    'description' => '',
+                ],
+            ],
+            'Fix'         => [
+                [
+                    'title'       => 'Fixed typo in vendor earning tooltip',
+                    'description' => '',
+                ],
+                [
+                    'title'       => 'Vendor was not getting a notification when order status change from cancelled to processing, on-hold, or completed. This has been fixed now',
+                    'description' => '',
+                ],
+            ],
+        ],
+    ],
+    [
+        'version'  => 'Version 3.2.15',
+        'released' => '2021-10-13',
+        'changes'  => [
+            'New Feature' => [
```

Comment:
It is not recommended to use uppercase letters and/or spaces in array keys. You could process it as needed while rendering on the frontend.
`'improvement'`, `'new_feature'`, these should be used as keys. And while rendering at frontend, you could convert them into `Improvement`, `New Feature`.